.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+tasks:
+  - init: go get -v -t -d ./...
```

Comment:
Is it possible to get the latest release ?
I'm not a fan of launching venom by bypassing the entire CI. Is it possible to run something like that:

```
LAST_RELEASE=$(curl -s https://api.github.com/repos/ovh/venom/releases | grep tag_name | head -n 1 | cut -d '"' -f 4) && \
DL=$(curl -s https://api.github.com/repos/ovh/venom/releases | grep ${LAST_RELEASE} | grep browser_download_url | grep -E 'linux-amd64' | cut -d '"' -f 4) &&
curl -L $DL -o venom
```